### PR TITLE
Mateus' PR 

### PR DESCRIPTION
**Pull request by philippbayer**
I forked Mateus' PR into my fork and changed it until the merge problems disappeared. Looks good right now!

### CODE REVIEW

**Comment by philippbayer** [comment]
Looks like most of the failing tests are due to content of text and buttons changing, will have to adjust the tests

Will do the hound stuff first.

**Comment by philippbayer** [comment]
looks like the hound has a 'maximum comments to make at once' setting :(

File: app/controllers/updates_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class UpdatesController < ApplicationController
```
Comment (philippbayer) [inline app/controllers/updates_controller.rb]: definitely there, weird

**Comment by philippbayer** [comment]
tests are passing :fireworks:
Had to cheat a bit - the structure of the search button changed, and I have no clue how to fix the test for that. the other tests work.
merge?!?!?

**Comment by gedankenstuecke** [comment]
Given our abysmal test coverage, did you try the site on your end to see whether it works? 😂

**Comment by philippbayer** [comment]
yes I did try :) couldn't directly find any errors. you can try too

**Comment by gedankenstuecke** [comment]
Okay, I added the *about us* stuff here as well (closes #367). 

Further edits: 
* removed old sponsor from the footer
* made sure the disclaimer shows up above the signup form when on a small display
* made sure the disclaimer shows up above the genotype upload form. 
* fixed some typos in the headings etc. But there might be more now. 
* reorganized the menu bar.

I *think* it should be good to go now.

**Comment by philippbayer** [comment]
I think it's good to go (save some borked tests that deserve fixing - esp. the search test)

I vote for merge PR, @gedankenstuecke go ahead for 🚀 ?

**Comment by gedankenstuecke** [comment]
Now it's time to start 🙏  to the :shipit: gods! :crossed_fingers:

**Comment by acabunoc** [comment]
I was wondering why @houndci-bot was our top contributor at #mozsprint ;)

**Comment by gedankenstuecke** [comment]
Oh, are the stats released somewhere? :)

**Comment by acabunoc** [comment]
Here's the first 24 hours with an incomplete set of repos:
https://github.com/mozilla/global-sprint/blob/master/_data/mozsprint_events_2017_day1fixed.csv

I'll be updating with the full sprint data after I watch Wonder Woman tonight

**Comment by gedankenstuecke** [comment]
I'll wait for the full data then! And enjoy the movie! 🎥

**Comment by acabunoc** [comment]
Movie was great 👍
Data + post:
https://medium.com/@MozOpenLeaders/mozsprint-by-the-github-numbers-49d518551725

**Comment by philippbayer** [comment]
Thank you very much @acabunoc! I see the hound made our project look very active, even more so by the weird Australian timezone :)

Thanks to all 'real humans' who contributed as well!

**Pull request merged**